### PR TITLE
Implement AMP + task reward

### PR DESCRIPTION
**Pull request by dtch1997**
So far we have used the pure discriminator reward in our AMP formulation, original paper also 'mixes' AMP and task rewards, which we can try to reproduce.

### CODE REVIEW

**Comment by dtch1997** [comment]
Target tasks under consideration (reference: https://arxiv.org/pdf/2104.02180.pdf): 

1. Target Location (go to designated position. Requires a command in the form of a designated goal position in the character's local coordinate frame.) 
2. Target Velocity (Target Heading in the paper, achieve target velocity in given direction. Requires a command in the form of a velocity vector which the policy attempts to match.) 
3. Target Heading (turn to face a particular direction.) 

It may be easiest to implement these as subclasses of `QuadrupedAMPBase` and override the reward function. To avoid duplicating many config files I may consider to have an all in one subclass that implements all types of command following. In addition, we can experiment with exporting a pre-trained discriminator trained in `QuadrupedAMP`.

**Comment by dtch1997** [comment]
At the moment we've started to integrate `TargetVelocity` into `QuadrupedAMPBase`. `reset` is known to run. Remaining todos: sanity-check the result of `reset`, and add observation / reward computation into the mix.

**Comment by dtch1997** [comment]
Okay everything runs. No guarantees that there's no bugs, but the first training run will tell us how things go.

**Comment by dtch1997** [comment]
Getup + locomotion seems good

https://user-images.githubusercontent.com/25474937/223241823-a52f99a5-69f8-4602-8dd5-73b7de4e00f8.mp4

**Comment by dtch1997** [comment]
Okay so now it's time for the large-scale experiments. I'm running sweeps on the value of `task.env.task.velErrorScale` to see which one does the best. Experiment's gonna last about 7h, will check back in tomorrow.

**Comment by dtch1997** [comment]
We're going to do a dataset of trotting motions only. We will additionally simplify the locomotion dataset to consist only of trotting at various speeds

**Comment by dtch1997** [comment]
Now we need some evaluation metrics to determine if the policy is responding to feedback control. To accomplish this I will implement some utilities to log the task command and the actuated state, as well as scripts to plot and visualize these quantities.

**Comment by dtch1997** [comment]
Evaluation metrics can be in a separate PR. Merging this in for now
